test-suite/js/src/test/scala/org/scalajs/testsuite/library/ObjectTest.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.Test
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{ JSBracketAccess, JSName }
+
+class ObjectTest {
+  import ObjectTest._
+
+  @Test def getOwnPropertyDescriptors(): Unit = {
+    assumeTrue(
+      "Assuming newer methods existence, which are not honored by Rhino",
+      Platform.executingInNodeJS && !Platform.executingInRhino)
```

Comment:
The indentation should be 4 spaces inside `()`, since it is a continuation line.

Also, `executingInNodeJS` already implies `!executingInRhino`. The the `&& !executingInRhino` is redundant (and even misleading, since there are other engines such as PhantomJS). Please keep only `Platform.executingInNodeJS`, and adapt the message accordingly.

This comment applies to the other tests, of course.